.github/workflows/ci_test.yml

```diff
@@ -94,3 +94,47 @@ jobs:
 #      with:
 #          name: documentation
 #          path: .tox/docs_out/
+  ci-tests-macos:
+    runs-on: macos-latest
```

Comment:
Maybe rename this to macos-latest-py3.11 or something like that so that the name is similar to the other tests.

Reply:
done.